VGMPlay/vgmspec171.txt

```diff
@@ -120,11 +120,13 @@ The format starts with a 256 byte header:
         It should be 0 if there us no YM2612 chip used.
         For version 1.01 and earlier files, the YM2413 clock rate should be
         used for the clock rate of the YM2612.
+        If bit 31 is set it is a YM3438.
```

Comment:
Perhaps this (and the other bits) should be for 1.71 (or 1.51) files and up only. Older VGM players may choke on this bit.

Reply:
I'd be fine with 1.51 and higher.

Reply:
Since 0.60 introduced the dual-chip flag on bit 30, I’d say these bits for sure get masked off starting with that version.

But is any action required here? Since it is in the vgm 1.71 spec, I’d think it is implicit…

Reply:
Just a note that the bit should not be set on VGMs with a <1.51 version should be enough.